src/Microsoft.DotNet.Build.Tasks/PackageFiles/executor/project.json

```diff
@@ -0,0 +1,26 @@
+{
+  "dependencies": {
+    "Microsoft.NETCore.TestHost": "1.0.0-rc2-23923",
```

Comment:
Why are you using these packages as opposed to making this a standard CLI application and using Microsoft.NETCore.App and dotnet.exe as the host?